src/index.d.ts

```diff
@@ -1,7 +1,7 @@
 // Import Third-party Dependencies
 import { Scanner } from "@nodesecure/scanner";
 
-export function extractAllAuthorsFromLibrary(library: Scanner.Payload, opts: options): Promise<authorResponse[]>
+export function extractAllAuthorsFromLibrary(library: Scanner.Payload, opts: options): Promise<authorResponse[] | []>
```

Comment:
That the same no? Nothing stop the Array to be empty.

Reply:
Hell yeah, Idk why I updated the typings 😅